amenity-points.mss

```diff
@@ -1555,6 +1555,7 @@
 /* Note that .text is also used in water.mss */
 .text-low-zoom[zoom < 10],
 .text[zoom >= 10] {
+  [feature = 'natural_cape'],
```

Comment:
I would make it exactly the same as islands. Big cape (like cape Florida, if it has a name) can be shown very early, but currently it has no limits and I think islands case nailed it, so you can just copy and paste the code, nothing fancy or unusual is needed.